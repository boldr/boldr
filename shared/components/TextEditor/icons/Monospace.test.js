import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Monospace from './Monospace';

describe('<Monospace />', () => {
  const wrapper = shallow(<Monospace />);
  it('renders <Monospace /> without breaking', () => {
    expect(wrapper.find('svg').length).toBe(1);
  });
  it('renders snapshot', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
