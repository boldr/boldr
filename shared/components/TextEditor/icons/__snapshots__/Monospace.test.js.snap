exports[`<Monospace /> renders snapshot 1`] = `
<svg
  height="18"
  viewBox="0 0 18 18"
  width="18"
  xmlns="http://www.w3.org/2000/svg">
  <path
    d="M6.559,5.023,9,12.242l2.428-7.219H14v9.953H12.035V12.256l.2-4.7L9.666,14.977H8.321L5.762,7.566l.2,4.689v2.721H4V5.023Z" />
  <polygon
    points="15 1 15 2 3 2 3 1 1 1 1 5 3 5 3 4 15 4 15 5 17 5 17 1 15 1" />
</svg>
`;
